onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -3456,10 +3456,52 @@ def aten_meshgrid(tensors: Sequence[TensorType]) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_min(self: TensorType) -> TensorType:
+@torch_op("aten::min")
+def aten_min(self: TReal) -> TReal:
     """min(Tensor self) -> Tensor"""
+    self_rank = op.Size(op.Shape(self))
+    if self_rank == 0:
```

Comment:
I remember Rama saying reduce ops can handle scalar inputs. Did we try this without the rank conditions?

Reply:
"scalar" means : 0 rank Tensor(scalar), or scalar something in in attribute?

Reply:
I meant 0 rank Tensor sorry

Reply:
works